templates/dash_template.jinja

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>{{page_title}}</title>
+        <link rel="icon" type="image/x-icon" href="img/favicon.png">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="css/styles.css">
+    </head>
+    <body>
+    <table>
+    <tr>
+        <th>Repository</th>
+        <th>Nightly Test</th>
+        <th>Nightly <br/> Benchmarks</th>
+        <th>Build Docs</th>
+        <th>Live Build</th>
+        <th>Other <br/> workflows</th>
+    </tr>
+
+    {% for project in all_projects %}
+    <tr>
+        <th><a href="{{project.repo_url}}">{{project.repo}}</a></th>
+        <th class="{{project.smoke_test.display_class}}"><a href="{{project.smoke_test.workflow_url}}">{{project.smoke_test.workflow_status}}</a></th>
```

Comment:
* Do we maybe want to use `<td>` tags for data cells (rather than `<th>`)?
* What happens in jinja for a project like rail_pipelines that doesn't have a smoke_test/project.smoke_test is None? Do we want to maybe want to do something like
```
{% if project.smoke_test is None %}
    <td></td>
{% else %}
    <td class="{{project.smoke_test.display_class}}"><a href="{{project.smoke_test.workflow_url}}">{{project.smoke_test.workflow_status}}</a></td>
{% endif %}
```

Reply:
Niiice!